src/convertToObject.js

```diff
@@ -13,6 +13,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const styles1 = sourceString.split(';');
+
+  const styles2 = styles1.map((elem) => elem.split(':'));
+
+  const styles3 = styles2.map((elem) => elem.map((item) =>
+    item.replace(/\s+/g, ' ')));
+
+  const styles4 = styles3.filter(elem => elem.length > 1);
+
+  const styles5 = styles4.map(elem => elem.map((item) =>
+    item.trim()));
```

Comment:
```suggestion
  const styles = sourceString.split(';')
    .map((elem) => elem.split(':'))
    .map((elem) => elem.map((item) => item.replace(/\s+/g, ' ')))
    .filter(elem => elem.length > 1)
    .map(elem => elem.map((item) => item.trim()));
```
Try not to name your variables like this. They have no meaning at all. If you don't want to give names to all those  variables you could use the chaining of methods as I've shown